components/layout_2020/flow/inline.rs

```diff
@@ -173,7 +173,7 @@ struct LineUnderConstruction {
     /// offset from `text-indent`.
     start_position: LogicalVec2<Length>,
 
-    /// The current inline position in the line being laid out into [`LineItems`] in this
+    /// The current inline position in the line being laid out into LineItems in this
```

Comment:
Converting [&#96;LineItems&#96;] to `LineItems` is not the correct solution here. This should be [&#96;LineItem&#96;]s  instead. For reference you can see some examples [here](https://github.com/servo/servo/blob/da609076c32abcc3d3267cb663dbae861b2dfba7/components/layout_2020/flow/inline.rs#L47)